### PR TITLE
Improve LightHouse Score

### PR DESCRIPTION
**Pull request by yonzilch**
I've changed several files.Here are the effects.

- Before
image

- After
image

Now,the only thing to improve LightHouse Score left is to set an image of favicon.png.
Fin!

### CODE REVIEW

File: templates/partials/header.html [hunk 1]
```diff
@@ -4,6 +4,7 @@
     <meta charset="UTF-8">
     <meta http-equiv="X-UA-Compatible" content="IE=edge">
     <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <meta name="description" content="This is an example description">
```
Comment (not-matthias) [inline templates/partials/header.html:7]: Can u remove this default description or make it configurable?

**Comment by not-matthias** [comment]
Thanks for the PR. I added a small comment, the rest can be merged after implementing the changes.

File: templates/partials/header.html (hunk 1 above)
Comment (yonzilch) [reply templates/partials/header.html]: I've make it configurable.Now just need to change the line in the `config.toml`
```
description = "This is an example description"
```

Comment (yonzilch) [reply templates/partials/header.html]: Please review this PR,as 2 months past,but no further process.Thanks!

**Comment by not-matthias** [comment]
Thanks!

**Pull request merged**